swagger-ui/dist/zipskee-v1.swagger.yaml

```diff
@@ -55,12 +55,8 @@ paths:
       tags:
        - Auth
       responses:
-        200:
-          description: ok
-        401:
-          description: unauthorized
-          schema:
-            $ref: '#/definitions/Errors'
+        204:
```

Comment:
Is the 204 a limitation of devise? The only reason I would keep this a 200 is so that we return a configurable "Logout" message or redirect that the client could consume.. thoughts?